agent/structs/acl_oss.go

```diff
@@ -2,6 +2,36 @@
 
 package structs
 
+import (
+	"fmt"
+
+	"github.com/hashicorp/consul/acl"
+)
+
 const (
 	EnterpriseACLPolicyGlobalManagement = ""
+
+	// aclPolicyTemplateServiceIdentity is the template used for synthesizing
+	// policies for service identities.
+	aclPolicyTemplateServiceIdentity = `
+service "%[1]s" {
```

Comment:
I have never seen this sprintf syntax before.

Reply:
It lets you use 1 parameter in multiple places without passing that parameter multiple times.